app/components/action-feed-entry-skeleton/template.hbs

```diff
@@ -0,0 +1,11 @@
+<div class="p-b-lg">
+    {{#content-placeholders as |placeholder|}}
+        <div class="icon--skeleton">{{placeholder.icon}}</div>
+    {{/content-placeholders}}
+    <div class="action-body--skeleton">
+        {{#content-placeholders as |placeholder|}}
```

Comment:
Could these two `{{#content-placeholders as |placeholder|}}` blocks be combined in one?